core/storage/user/gae_models.py

```diff
@@ -288,6 +288,27 @@ class UserContributionsModel(base_models.BaseModel):
     edited_exploration_ids = ndb.StringProperty(
         repeated=True, indexed=True, default=None)
 
+    @staticmethod
+    def export_data(user_id):
+        """(Takeout) Export user-relevant properties of UserContributionsModel.
+
+        Args:
+            user_id: str. The user_id denotes which user's data to extract.
+
+        Returns:
+            dict or None. The user-relevant properties of UserContributionsModel
```

Comment:
This description has some weird sentence fragments (e.g. the last sentence). Suggest:

dict or None. A dict containing the user-relevant properties of UserContributionsModel (i.e. the IDs of created and edited explorations), or None if the user_id is invalid.

Reply:
Ack

Reply:
Sorry, still awkward. The first sentence implies that this function always returns stuff in a python dict format, which is not the case.

Reply:
My bad, should be fixed now.